src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.SkeletonReferenceCache.cs

```diff
@@ -241,11 +241,7 @@ private bool TryReadSkeletonReferenceSetAtThisVersion(VersionStamp version, out
                 {
                     using var stream = SerializableBytes.CreateWritableStream();
 
-                    var optionsService = workspace.Services.GetService<IWorkspaceConfigurationService>();
-                    var doNotClone = optionsService != null && optionsService.Options.DisableCloneWhenProducingSkeletonReferences;
-
-                    var compilationToEmit = doNotClone ? compilation : compilation.Clone();
```

Comment:
Does anybody call Clone now? Does the compiler want to obsolete it?

Reply:
we still have it in one place (doc comments).  but i'll look to see if we can remove that too.